Formula/b/bold.rb

```diff
@@ -0,0 +1,48 @@
+class Bold < Formula
+  desc "Drop-in replacement for Apple system linker ld"
+  homepage "https://github.com/kubkon/bold"
+  url "https://github.com/kubkon/bold/archive/refs/tags/v0.1.0.tar.gz"
+  sha256 "2496f04e47c9d5e17ef273b26519abf429b5c3e3df6d264f2941735088253ec0"
+  license "MIT"
+
+  depends_on "zig" => :build
+  depends_on :macos # does not build on linux
+
+  def install
+    # Fix illegal instruction errors when using bottles on older CPUs.
+    # https://github.com/Homebrew/homebrew-core/issues/92282
+    cpu = case Hardware.oldest_cpu
+    when :arm_vortex_tempest then "apple_m1" # See `zig targets`.
+    else Hardware.oldest_cpu
+    end
+
```

Comment:
I think instead of adding this to every zig formula this should be a Homebrew/brew shim.

Reply:
agree, we should have some zig shim. 👍

Reply:
Maybe not a shim; just `std_zig_args`.